src/garage/torch/utils.py

```diff
@@ -130,3 +130,43 @@ def global_device():
     # pylint: disable=global-statement
     global _DEVICE
     return _DEVICE
+
+
+def from_numpy(*args):
```

Comment:
please coordinate with Avnish -- i think these appear in one of his PRs as well.